### DISCUSSION TITLE
Open a specific slide from link on different webpage

### DISCUSSION THREAD

**Discussion by voltmer1** [biati-digital/glightbox#463] [Q&A]
It would be very helpful to be able to share a link that would open a slideshow to a particular slide. Is this possible currently, or is it possible to add it as a feature in a future version?

**Comment by biati-digital** [comment]
Probably this will never be added directly in the library as this is the first time this functionality has been requested in years.

The functionality you need it's really easy to implement, the API already allows you to open the Lightbox in a specific slide, you just need to get some query parameters from the URL and call the API. Checkout the documentation to learn about the [API](https://github.com/biati-digital/glightbox#api) and also check [How to obtain the query string from the current URL with JavaScript?](https://stackoverflow.com/questions/9870512/how-to-obtain-the-query-string-from-the-current-url-with-javascript)
